packages/es-components/src/components/containers/sliding-pane/SlidingPane.tsx

```diff
@@ -11,7 +11,37 @@ import { useDisableBodyScroll } from '../../util/useDisableBodyScroll';
 import { useRootNodeLocator } from '../../util/useRootNode';
 import { useTheme } from '../../util/useTheme';
 
-const PaneBase = styled(Modal)`
+interface PaneBaseProps {
+  paneWidth: string;
+}
+
+interface CloseLinkProps {
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+interface SlidingPaneProps {
```

Comment:
Since we're spreading props into Modal, our Type should cover that as well.

```suggestion
interface SlidingPaneProps extends Parameters<Modal>[0] {
```